packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1619,4 +1619,54 @@ void main() {
     expect((mergeableMaterial.children.first as MaterialSlice).color, firstPanelColor);
     expect((mergeableMaterial.children.last as MaterialSlice).color, secondPanelColor);
   });
+
+  testWidgets('default materialGapSize should be 16', (WidgetTester tester) async {
```

Comment:
nit: `ExpansionPanelList.materialGapSize defaults to 16.0`

Reply:
done 👍 o